functions.go

```diff
@@ -410,17 +420,12 @@ func (f *Function) RemoveInstance(isSync bool) (string, error) {
 	)
 
 	f.OnceAddInstance = new(sync.Once)
-
-	if orch.GetSnapshotsEnabled() {
-		f.OffloadInstance()
-		r = "Successfully offloaded instance " + f.vmID
+	
+	if isSync {
+		err = orch.StopSingleVM(context.Background(), f.vmID)
 	} else {
-		if isSync {
```

Comment:
please document this change in the PR's desc

Reply:
Actually, I already mentioned it:

> * Dropped VM offloading APIs and replaced them with snapshot creation or VM shutdown;